dotnet/src/SemanticKernel.Abstractions/SkillDefinition/ISKFunction.cs

```diff
@@ -59,20 +58,6 @@ public interface ISKFunction
         CompleteRequestSettings? settings = null,
         CancellationToken cancellationToken = default);
 
-    /// <summary>
-    /// Invoke the <see cref="ISKFunction"/>.
-    /// </summary>
-    /// <param name="input">String input</param>
-    /// <param name="settings">LLM completion settings (for semantic functions only)</param>
-    /// <param name="logger">Application logger</param>
-    /// <param name="cancellationToken">The <see cref="CancellationToken"/> to monitor for cancellation requests. The default is <see cref="CancellationToken.None"/>.</param>
-    /// <returns>The updated context, potentially a new one if context switching is implemented.</returns>
-    Task<SKContext> InvokeAsync(
-        string? input = null,
-        CompleteRequestSettings? settings = null,
-        ILogger? logger = null,
-        CancellationToken cancellationToken = default);
```

Comment:
Is it worth having a CHANGELOG.md file in the repo where we list the breaking changes for each release?

Format could be something like this:

## Next Release

### New features:

-   ...

### Breaking changes:

- Removed `ISKFunction.InvokeAsync(string, CompletionRequestSettings, ILogger, CancellationToken)`, use `ISKFunction.InvokeAsync(string, CompletionRequestSettings, CancellationToken)` instead

## Release 0.18.230725.3-preview

### New features:

-   ...
-   ...

### Breaking changes:

- ...
- ...

Reply:
I'm planning on putting the breaking changes section in the Releases page. That doesn't have the "next section" though. It's a good idea -- let's talk offline and sync across kernel languages

Reply:
This breaking change broke one of our samples. https://github.com/microsoft/semantic-kernel/issues/2361